### DISCUSSION TITLE
Help me find the right transceiver

### DISCUSSION THREAD

**Discussion by lasryaric** [jgromes/RadioLib#284] [General]
Hi,
I bough a few C1101 transceivers  on amazon (here: https://www.amazon.com/gp/product/B00U5TO37W/ref=ppx_yo_dt_b_asin_title_o02_s00?ie=UTF8&psc=1 ), and only 2 of them work, and I have to push really hard on the connection pins, otherwise I never receive the data I am sending. If I push hard on the pins, then I receive my data 100% of the time.

Can you recommend a low frequency transceiver component that I can buy, that is not too expensive and reliable?
I need a low frequency one (~400 mhz) because I want to transmit under water.

Thanks.

**Comment by jgromes** [comment]
Sorry, I don't have any experience with underwater transmissions, but I suspect 400 MHz will still be quite high. In any case, SX1276/78 is a versatile module, it can transmit FSK and LoRa in several bands in the range 150 - 915 MHz (please check your local laws for allowed frequency bands). What range will you get underwater - no idea about that. Please let me know though!
